carmm/analyse/counterpoise_onepot.py

```diff
@@ -0,0 +1,53 @@
+def create_counterpoise_input(complex_struc=None, a_id=None, b_id=None, symbol_not_index=True):
+    """
+    This function creates all the geometry.in files in separate folders, assuming a binding complex
+    AB.  
+    Parameters:
+        complex_struc: string
+            This is the filename of the ase trajectory file (or any file type ase.io.read accepts)
+            which stores the optimized structure of the binding complex
+        a_id: list of atom symbols or atom indices for species A
+        b_id: list of atom symbols or atom indices for species B
+            Please use both symbols or both indices for a_id and b_id.
+        symbol_not_index: bool
+            This indicates whether symbols are used.
+    """
+    import subprocess
+    from ase.io import read, write
+    from ase.io.aims import write_aims
+    # Let's say we have A and B in this complex
+    # ?_only has A or B in the geometry of the binding complex with its own basis
+    # ?_ghost has A or B in the same geometry with ghost atoms added
+    dir_list = ['A_only', 'A_ghost', 'B_only', 'B_ghost']
+    subprocess.run(['mkdir'] + dir_list)
```

Comment:
`os.makedirs()` has a way of handling existing errors - what is the outcome if any of the folders exist when subprocess.run(`mkdir dir`) is used if `dir `already exists?

Reply:
It does complain about file exists, giving a message like "mkdir: cannot create directory ‘A_only’: File exists". But the program would proceed and geometry.in files would be created and copied correctly.